pkg/render/common/networkpolicy/networkpolicy.go

```diff
@@ -279,6 +281,10 @@ func (h *NetworkPolicyHelper) ManagerSourceEntityRule() v3.EntityRule {
 	return CreateSourceEntityRule(h.namespace("tigera-manager"), "tigera-manager")
 }
 
+func (h *NetworkPolicyHelper) APIServerSourceEntityRule(v operatorv1.ProductVariant) v3.EntityRule {
+	return CreateSourceEntityRule(h.namespace(meta.APIServerNamespace(v)), "tigera-apiserver")
```

Comment:
Also the deployment name itself is different. It's called calico-apiserver in OSS. See apiserver.go.